### ISSUE TITLE
模型训练的val_loss不断升高

### ISSUE THREAD

**Issue by T0L0ve** [shenweichen/DeepCTR#514]
**Describe the question(问题描述)**
使用DeepFEFM训练模型，训练集loss在下降，val_loss第一次最低然后逐渐升高。按理说这种情况是模型过拟合了，但训练第一个epoch就过拟合有些不合理吧

**Additional context**

![image](https://user-images.githubusercontent.com/38580985/225185762-d7442394-19c5-4bcd-970a-93487329ac18.png)

**Operating environment(运行环境):**
 - python version [e.g. 3.6]
 - tensorflow version [e.g. 1.4.0, 1.15.0, 2.10.0]
 - deepctr version [e.g. 0.9.2,]

**Comment by amhml** [comment]
数据太少了，多搞点数据

**Comment by shenweichen** [comment]
从描述看，训练集 loss 下降但 val_loss 从第一个 epoch 后升高，通常是数据量、验证集划分、过拟合或分布偏移问题，不是 DeepCTR 模型代码本身的特定 bug。

建议检查样本量、特征泄漏、train/validation 分布、正则化、dropout、early stopping 和学习率。Closing as answered.
